click_odoo_contrib/_dbutils.py

```diff
@@ -12,7 +13,11 @@
 def pg_connect():
     conn = odoo.sql_db.db_connect("postgres")
     cr = conn.cursor()
-    cr.autocommit(True)
```

Comment:
reset_config_parameters seems to use `env["ir.config_parameter"].init(force=True)` so is this statement about not using the ORM really true?